dandi/cli/tests/data/update_dandiset_from_doi/biorxiv.json

```diff
@@ -13,13 +13,13 @@
     ],
     "version": "draft",
     "@context": "https://raw.githubusercontent.com/dandi/schema/master/releases/0.6.4/context.json",
-    "citation": "Aguillon-Rodriguez, Valeria; Angelaki, Dora E.; Bayer, Hannah M.; Bonacchi, Niccol\u00f2; Carandini, Matteo; Cazettes, Fanny; Chapuis, Gaelle A.; Churchland, Anne K.; Dan, Yang; Dewitt, Eric E. J.; Faulkner, Mayo; Forrest, Hamish; Haetzel, Laura M.; Hausser, Michael; Hofer, Sonja B.; Hu, Fei; Khanal, Anup; Krasniak, Christopher S.; Laranjeira, In\u00eas; Mainen, Zachary F.; Meijer, Guido T.; Miska, Nathaniel J.; Mrsic-Flogel, Thomas D.; Murakami, Masayoshi; Noel, Jean-Paul; Pan-Vazquez, Alejandro; Rossant, Cyrille; Sanders, Joshua I.; Socha, Karolina Z.; Terry, Rebecca; Urai, Anne E.; Vergara, Hernando M.; Wells, Miles J.; Wilson, Christian J.; Witten, Ilana B.; Wool, Lauren E.; Zador, Anthony (2023) Standardized and reproducible measurement of decision-making in mice (Version draft) [Data set]. DANDI archive. http://localhost:8085/dandiset/000001/draft",
+    "citation": "Aguillon-Rodriguez, Valeria; Angelaki, Dora E.; Bayer, Hannah M.; Bonacchi, Niccol\u00f2; Carandini, Matteo; Cazettes, Fanny; Chapuis, Gaelle A.; Churchland, Anne K.; Dan, Yang; Dewitt, Eric E. J.; Faulkner, Mayo; Forrest, Hamish; Haetzel, Laura M.; Hausser, Michael; Hofer, Sonja B.; Hu, Fei; Khanal, Anup; Krasniak, Christopher S.; Laranjeira, In\u00eas; Mainen, Zachary F.; Meijer, Guido T.; Miska, Nathaniel J.; Mrsic-Flogel, Thomas D.; Murakami, Masayoshi; Noel, Jean-Paul; Pan-Vazquez, Alejandro; Rossant, Cyrille; Sanders, Joshua I.; Socha, Karolina Z.; Terry, Rebecca; Urai, Anne E.; Vergara, Hernando M.; Wells, Miles J.; Wilson, Christian J.; Witten, Ilana B.; Wool, Lauren E.; Zador, Anthony (2023) Standardized and reproducible measurement of decision-making in mice (Version draft) [Data set]. DANDI Archive. http://localhost:8085/dandiset/000001/draft",
```

Comment:
those are coming from dandi-archive

```
❯ git grep 'DANDI archive'
dandiapi/api/models/version.py:        citation = f'{name} ({year}). (Version {version}) [Data set]. DANDI archive. {url}'
dandiapi/api/models/version.py:                    f'{cl} ({year}) {name} (Version {version}) [Data set]. DANDI archive. {url}'
...
```

there should be a "satellite" PR there to likely preceed these changes or we should improve testing here to allow for flexibility (didn't look ATM) but these are the cause of failing tests ATM e.g. https://github.com/dandi/dandi-cli/actions/runs/12262974763/job/34213512790?pr=1548#step:8:25905

Reply:
do you think you could remove those changes in the tests for now so we could not delay accepting major portion of the changes @kabilar to not breed conflicts etc?

Reply:
Hi @yarikoptic, I just created https://github.com/dandi/dandi-archive/pull/2114.

Reply:
it was merged now, so we should be all set here.  I will retrigger tests to see if all good (frankly I don't remember if we need release of archive first) and then we could proceed